### ISSUE TITLE
Group bindings under a subpane/group

### ISSUE THREAD

**Issue by minitoine** [cocopon/tweakpane#567]
Hello :)

I was wondering if it is possible to add a 'SubPane' (or group?) component to the library to virtually (no visual impact) group multiple inputs together like folders (but without header/left padding).
This could be useful when dynamically changing inputs. Instead of hidding/showing each inputs of interest, only actioning the group object would be needed.

Thaaanks :)

**Comment by aczw** [comment]
I agree with this issue, but as a workaround I've manually "grouped" all related blades by putting them in an array. This way I can call

```ts
groupOfBlades.forEach((blade) => (blade.hidden = true));
```
